### ISSUE TITLE
Permissions on install too restrictive

### ISSUE THREAD

**Issue by phreed** [gdubw/gng#18]
This is on CentOS 7.

I used the installation procedure described in `README.md` 
```
# Downloading the latest release
 curl -fsSL https://github.com/gdubw/gng/releases/latest/download/gng-installer.sh -o gng-installer.sh
 sudo bash ./gng-installer.sh
 ```
This resulted in the `/opt/gng` directory as well as several symbolic links being created with 700 permissions.
This makes gng protected from normal users.

 I did a `chmod 755 -R /opt/gng` and created an alternative.
 ```
 sudo alternatives --install /usr/bin/gw gw /opt/gng/bin/gng 1
 ```
 It now seems to be working correctly.

**Comment by dantesun** [comment]
related to #15

**Comment by dantesun** [comment]
I forgot the set the umask, fixed in master branch now.

**Comment by dantesun** [comment]
fixed in v1.0.3 https://github.com/gdubw/gng/releases/download/v1.0.3/gng-installer.sh
